model/metrics/container.yaml

```
@@ -0,0 +1,57 @@
+groups:
+  - id: metric.container.cpu.usage
+    type: metric
+    metric_name: container.cpu.usage
+    brief: "Recent CPU utilization for the container."
+    note: >
+      CPU usage percentage normalized by the number of CPU cores.
+      The value range is [0.0,1.0].
+    instrument: gauge
+    unit: "1"
+  - id: metric.container.memory.usage
+    type: metric
+    metric_name: container.memory.usage
+    brief: "Recent memory utilization for the container."
+    note: >
+      Memory usage percentage.
+      The value range is [0.0,1.0].
+    instrument: gauge
+    unit: "1"
+  - id: metric.container.disk.read.bytes
+    type: metric
+    metric_name: container.disk.read.bytes
```

check out #260, existing metric names seems to be more aligned with

```suggestion
    metric_name: container.disk.read.io
```

---

oh ya, and then there was a discussion previously about not having separate read/write metrics but instead making those dimensions, so really more like:

```suggestion
    metric_name: container.disk.io
```